components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -14,46 +14,88 @@ toc = true
+To avoid configuring DNS for your test instance, we recommend adding an entry to your workstation's `/etc/hosts` file.
+Use the following command to append the required entry:
+
+```bash
+echo 192.168.33.199 chef-automate.test | sudo tee -a /etc/hosts
+```
+
 ## Create a Vagrantfile
 
 Copy and paste the command below to create a `Vagrantfile`:
 
 ```ruby
-cat > Vagrantfile <<EOH
-# Put the contents of this snippet in a file named Vagrantfile
+cat > Vagrantfile <<'EOH'
+CFG_BOX      = 'bento/ubuntu-18.04'
+CFG_IP       = '192.168.33.199'
+CFG_HOSTNAME = 'chef-automate.test'
+
+$deployscript = <<-SCRIPT
+export DEBIAN_FRONTEND='noninteractive'
+apt-get -qq update
+apt-get install -y --no-install-recommends curl unzip
+apt-get clean
+sysctl -w vm.max_map_count=262144
+sysctl -w vm.dirty_expire_centisecs=20000
+echo "${CFG_IP} ${CFG_HOSTNAME}" | tee -a /etc/hosts
+curl -fsSL https://packages.chef.io/files/current/automate/latest/chef-automate_linux_amd64.zip -o /tmp/chef-automate_linux_amd64.zip
+unzip -qod /usr/local/bin /tmp/chef-automate_linux_amd64.zip
+chmod +x /usr/local/bin/chef-automate
+chef-automate deploy --accept-terms-and-mlsa
+echo "Server is up and running. Please log in at https://${CFG_HOSTNAME}/"
+echo 'You may log in using credentials provided below:'
+cat /home/vagrant/automate-credentials.toml
+SCRIPT
+
 Vagrant.configure(2) do |config|
+  class AcceptLicense
+    def to_s
+      return 'true' if ENV['ACCEPT_CHEF_TERMS_AND_MLSA'] == 'true'
+      puts <<TERMS
+To continue, you'll need to accept our terms of service:
+Terms of Service
+https://www.chef.io/terms-of-service
+
+Master License and Services Agreement
+https://www.chef.io/online-master-agreement
+
+I agree to the Terms of Service and the Master License and Services Agreement (y/n)
+TERMS
+      if STDIN.gets.chomp == 'y'
+        'true'
+      else
+        puts 'Chef Software Terms of Service and Master License and Services Agreement were not accepted'
+        exit 1
+      end
+    end
+  end
+  config.vm.box      = CFG_BOX
+  config.vm.hostname = CFG_HOSTNAME
+
   config.vm.provider "virtualbox" do |v|
-    v.memory = 8192
-    v.cpus = 2
+    v.name       = 'chef-automate'
+    v.memory     = 4096
+    v.cpus       = 2
+    v.customize ['modifyvm', :id, '--audio', 'none']
   end
 
-  config.vm.box = "bento/ubuntu-16.04"
-  config.vm.synced_folder ".", "/opt/a2-testing", create: true
-  config.vm.hostname = 'chef-automate.test'
-  config.vm.network 'private_network', ip: '192.168.33.199'
-  config.vm.provision "shell", inline: "apt-get update && apt-get install -y unzip"
-  config.vm.provision "shell", inline: "sysctl -w vm.max_map_count=262144"
-  config.vm.provision "shell", inline: "sysctl -w vm.dirty_expire_centisecs=20000"
+  config.vm.synced_folder '.', '/opt/a2-testing', create: true
+  config.vm.network       'private_network', ip: CFG_IP
+  config.vm.provision     'shell', env: {'CFG_IP' => CFG_IP,
```

Comment:
Can we make sure we're doing the equivalent of `config.vm.provision "shell", inline: "apt-get update && apt-get install -y unzip`. ?

Reply:
Ah, I see it now.